examples/stock/company.vy

```diff
@@ -44,7 +44,7 @@ def buyStock():
     buy_order: uint256(currency_value) = msg.value / self.price # rounds down
 
     # Check that there are enough shares to buy.
-    assert self.stockAvailable() >= buy_order
+    assert self.holdings[self.company] >= buy_order
```

Comment:
might be cleaner to refactor into a public/private pair: private `_stockAvailable` and public `stockAvailable` (which simply calls `_stockAvailable`), that way the code can be reused.

Reply:
Made the change and updated the documentation accordingly.